src/pages/ComponentsPage/ComponentsPage.sass

```diff
@@ -1,3 +1,89 @@
 @import '../../styles/core'
 
 .ComponentsPage
+    #ComponentsList
```

Comment:
Also worth noting: pages are meant as grids that contain components, so generally, pages should be pretty bare in terms of CSS.